onnxruntime/test/python/transformers/benchmark_mha.py

```diff
@@ -441,7 +441,7 @@ def run_tflops_test(use_gpu: bool = True, enable_cuda_graph: bool = False, repea
                     sequence_length=sequence_length,
                     num_heads=num_heads,
                     head_size=head_size,
-                    causal=True,
+                    causal=causal,
```

Comment:
Is it intended? `causal` is hardcoded as False in Ln434.

Reply:
Yes. Causal mask is not supported in current FlashAttention implementation. Setting it to `True` means we can't test FlashAttention.